frontend/src/scenes/error-tracking/ErrorTrackingScene.tsx

```diff
@@ -64,58 +61,57 @@ export function ErrorTrackingScene(): JSX.Element {
-                    ) : (
-                        <IngestionStatusCheck />
-                    )}
-                    <ErrorTrackingFilters />
-                    <LemonDivider className="mt-2" />
-                    <ErrorTrackingListOptions />
-                    <Query query={query} context={context} />
-                </BindLogic>
-            </ErrorTrackingSetupPrompt>
-        </FloatingContainerContext.Provider>
+        <ErrorTrackingSetupPrompt>
+            <BindLogic logic={errorTrackingDataNodeLogic} props={{ key: insightVizDataNodeKey(insightProps) }}>
+                <Header />
+                {hasSentExceptionEventLoading ? null : hasSentExceptionEvent ? (
+                    <FeedbackNotice text="Error tracking is currently in beta. Thanks for taking part! We'd love to hear what you think." />
+                ) : (
+                    <IngestionStatusCheck />
+                )}
+                <ErrorTrackingFilters />
+                <LemonDivider className="mt-2" />
+                <ErrorTrackingListOptions />
+                <Query query={query} context={context} />
+            </BindLogic>
+        </ErrorTrackingSetupPrompt>
     )
 }
 
 const VolumeColumn: QueryContextColumnComponent = (props) => {
+    const { dateRange } = useValues(errorTrackingLogic)
+    const { sparklineSelectedPeriod } = useValues(errorTrackingSceneLogic)
     const record = props.record as ErrorTrackingIssue
-    const [values, unit, interval] = useSparklineData(record.aggregations)
+    const { values, labels } = useSparklineData(sparklineSelectedPeriod, dateRange, record.aggregations)
     return (
         <div className="flex justify-end">
-            <OccurrenceSparkline className="h-8" unit={unit} interval={interval} displayXAxis={false} values={values} />
+            <OccurrenceSparkline className="h-8" values={values} labels={labels} displayXAxis={false} />
         </div>
     )
 }
 
 const VolumeColumnHeader: QueryContextColumnTitleComponent = ({ columnName }) => {
-    const { sparklineSelectedPeriod, sparklineOptions } = useValues(errorTrackingLogic)
-    const { setSparklineSelectedPeriod: onChange } = useActions(errorTrackingLogic)
+    const { sparklineSelectedPeriod, sparklineOptions } = useValues(errorTrackingSceneLogic)
+    const { setSparklineSelectedPeriod } = useActions(errorTrackingSceneLogic)
 
-    return sparklineSelectedPeriod && sparklineOptions ? (
+    return (
         <div className="flex justify-between items-center min-w-64">
             <div>{columnName}</div>
             <LemonSegmentedButton
                 size="xsmall"
                 value={sparklineSelectedPeriod}
-                options={Object.values(sparklineOptions)}
-                onChange={onChange}
+                options={sparklineOptions}
+                onChange={(newValue) => setSparklineSelectedPeriod(newValue)}
```

Comment:
Any reason not to do this?
```suggestion
                onChange={setSparklineSelectedPeriod}
```